p2p/proto/consensus.proto

```diff
@@ -1,12 +1,30 @@
 syntax = "proto3";
 
+import "p2p/proto/class.proto";
 import "p2p/proto/common.proto";
 import "p2p/proto/transaction.proto";
 
 // WIP - will change
 
+// like MempoolTransaction, except it has a L1Handler variant.
+message ConsensusTransaction {
+    message DeclareV3 {
+        DeclareV3Common common = 1;
+        Cairo1Class class = 2;
```

Comment:
Why are we defining DeclareV3 inside of ConsensusTx and MempoolTx? Shouldn't this be defined once and used by both?

This probably makes a bigger difference in rust, since there we'll have functionality for the type, but I figured the structure here would also mirror that.

Reply:
Fixed
<!-- Reviewable comment -OBtcpuN5jr8tOW5VkIN:b-886gsx -->
<!-- Sent from Reviewable.io -->